assets/js/SignGroupExpiration.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import dateFormat from 'dateformat';
+import { SignGroup } from './types';
+
+export default function SignGroupExpiration({ group }: { group: SignGroup }) {
```

Comment:
Since this is a public module now, maybe add a little test for it? Also, what do you think about naming it `SignGroupExpirationDescription`? From the name I was expecting it do something about the expiration, rather than just describing it. (For that matter, it might even work as a non-react function that returns the expiration string inside the `.sign_groups--group-expiration`? Not sure, there might be some value in keeping the structure in one place, too.)

Reply:
Good points all around. I think I will keep it as a component since the markup is relevant.